modules/network-observability-operator-install.adoc

```diff
@@ -32,15 +32,15 @@ This documentation assumes that your `LokiStack` instance name is `loki`. Using
 ** Set the *type* to `KAFKA`.
 ** Set the *address* as `kafka-cluster-kafka-bootstrap.netobserv`.
 ** Set the *topic* as `netobserv-flows-export`. The Operator exports all flows to the configured Kafka topic.
-** Set the following *tls* specifications:
-*** *certFile*: `service-ca.crt`, *name*: `kafka-gateway-ca-bundle`, and *type*: `configmap`.
-+
-You can also configure this option at a later time by directly editing the YAML. For more information, see _Export enriched network flow data_.
-* *loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`. The first part of the URL, "loki", should match the name of your LokiStack.
-* *loki.statusUrl*: Set this to `https://loki-query-frontend-http.netobserv.svc:3100/`. The first part of the URL, "loki", should match the name of your LokiStack.
-* *loki.authToken*: Select the `FORWARD` value.
-* *tls.enable*: Verify that the box is checked so it is enabled.
-* *statusTls*: The `enable` value is false by default.
+//** Set the following *tls* specifications:
+//*** *certFile*: `service-ca.crt`, *name*: `kafka-gateway-ca-bundle`, and *type*: `configmap`.
+//+
+//You can also configure this option at a later time by directly editing the YAML. For more information, see _Export enriched network flow data_.
+//* *loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`. The first part of the URL, "loki", should match the name of your LokiStack.
```

Comment:
This kinda ties in with the Lokiless NetObserv docs PR [comments](https://github.com/openshift/openshift-docs/pull/62901/files#r1296835041), but I was thinking that `loki.url` was not going to be part of the CRD after 1.4. Maybe I'm misunderstanding this again :/

Reply:
@jpinsonneau

Reply:
yes using `mode: LOKISTACK` you will only need `loki.enable: true` to use loki and `lokiStack.name` + `lokiStack.namespace` if not using the default values to get the related loki-operator instance.

As we only support loki-operator, I don't think we need to go in details about manual mode but FYI all the previous values will be moved to that section as:
```
  loki:
    mode: MANUAL
    manual:
      ingesterUrl: 'https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network/'
      statusUrl: 'https://loki-query-frontend-http.netobserv.svc:3100/'
      authToken: FORWARD
      tls:
        enable: true
        caCert:
          type: configmap
          name: loki-gateway-ca-bundle
          certFile: service-ca.crt
        insecureSkipVerify: false
      statusTls:
        enable: true
        caCert:
          certFile: service-ca.crt
          name: loki-ca-bundle
          type: configmap
        insecureSkipVerify: false
        userCert:
          certFile: tls.crt
          certKey: tls.key
          name: loki-query-frontend-http
          type: secret
```

Reply:
Ahh okay, I see thank you for explaining that the `MANUAL` mode gets all the previous values. I agree we don't need to go into manual mode details here.

Reply:
I reckon we'll continue to use `MANUAL` mode in the QE Perf automation, unless we want to switch it over to `LOKISTACK` for testing purposes - @memodi @Amoghrd any thoughts around that?

Reply:
@nathan-weinberg I'm waiting to know if https://issues.redhat.com/browse/NETOBSERV-764 will go in with 1.4 or if we'll push to a later release because I think `LOKISTACK` mode is encapsulated in the PR for that story. So I guess, the documentation will use `MANUAL` mode unless the Improve LokiStack integration PR ends up getting merged. @acmenezes is going to let me know

Reply:
@nathan-weinberg - why would we use `MANUAL` mode in QE Perf automation? TBH I am hearing about these modes for the first time, so please point me to any resources for me to read about them.

Reply:
@nathan-weinberg the CRD will change anyways so I don't know if it will be worth using `manual` before moving to `lokistack` mode or directly update everything in your tests.
If you anyways create your `FlowCollector` using a previous API version (let's say `v1beta1`), we will migrate it to the latest (`v1beta2`) using webhooks taking care of setting you in `manual` mode with the proper bindings :smile_cat: 

@skrthomas @memodi this epic [was postponed to 1.5 -- see target version](https://issues.redhat.com/browse/NETOBSERV-763).
It's mainly pending for this PR if you wanna see all impacts: https://github.com/netobserv/network-observability-operator/pull/329

Reply:
@nathan-weinberg @jpinsonneau @memodi Yes, I think that now the only piece that's available to customers is the rate-limit info. As Julien mentioned in our synch call, this code is already merged and available with Loki 5.6+. I'll relocate these CRD updates to a draft PR and save them for 1.5 timeframe. Thanks for the clarifications.